x-pack/plugins/alerting/server/routes/get_rule_execution_log.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from 'kibana/server';
+import { schema } from '@kbn/config-schema';
+import { ILicenseState } from '../lib';
+import { GetExecutionLogByIdParams } from '../rules_client';
+import { RewriteRequestCase, verifyAccessAndContext } from './lib';
+import { AlertingRequestHandlerContext, INTERNAL_BASE_ALERTING_API_PATH } from '../types';
+
+const paramSchema = schema.object({
+  id: schema.string(),
+});
+
+const querySchema = schema.object({
+  date_start: schema.string(),
+  date_end: schema.maybe(schema.string()),
+  filter: schema.maybe(schema.string()),
```

Comment:
I am wondering if we can use this type for filter to just be `{ bool: estypes.QueryDslBoolQuery }` from `import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';`. I think it will be better to use something like that to avoid our UI to build our filter filed in KQL way. Maybe something we can investigate later.

Reply:
With the event log aggregation, I am reusing existing logic for the filter that assumes KQL filter string, so maybe we can leave this as is for now and create a followup issue for this.

Reply:
++